### PR TITLE
Merge request fixed #55 invalid @depends tag

### LINKED ISSUE #55: Invalid phpdoc tag for @depends annotation
'@depends' tags of PHPUnit is not valid as phpdoc tag. This tag should be treated as valid tag.

Code snippet of test functions.
```
` /**
   * @depends test_add_entry
   */
    public function test_delete_entry() {
        $this->resetAfterTest(true);
        $entryid = $this->create_entry();
        tool_sumitnegi\api::remove($entryid);
        $entry = tool_sumitnegi\api::get($entryid);
        $this->assertEmpty($entry, 'Got data for entryid');
    }`
```

Error on code checking:
image

### PR DESCRIPTION
**Pull request by sumitnegi933**

### CODE REVIEW

File: file.php [hunk 1]
```diff
@@ -838,6 +838,7 @@ class local_moodlecheck_phpdocs {
         'expectedExceptionMessage',
         'expectedExceptionMessageRegExp',
         'group',
+        'depends',// PHPUnit tags.
```
Comment (kabalin) [inline file.php:841]: Comment is not needed, there are many annotations related to phpunit there.

**Comment by kabalin** [comment]
Hello sumitnegi933, thanks for contribution 👍 . Although, I an not maintainer of this plugin, I thought I would review it anyway, to simplify the work for maintainers. See the inline comment I added above. Also commit message can be improved, just state something like "Add support for 'depends' annotation". Stating issue number is totally optional, but if you really want to mention it, add `fixes #55` to commit message body 😉

**Comment by sumitnegi933** [comment]
Thanks Ruslan,
Yes, your suggestion is very useful, I will make it in that way.

**Comment by sumitnegi933** [comment]
The idea behind the mentioning issue number was that it will show there with the issue page and if someone wants to see what fixes has been done for the issue then he can go to commit and see.

**Comment by kabalin** [comment]
> The idea behind the mentioning issue number was that it will show there with the issue page and if someone wants to see what fixes has been done for the issue then he can go to commit and see.

That is true. But when you close the issue you may say "Fixed in <commit hash>" or "Fixed in #56 " to achieve for same effect. It is up to you, add it to commit message if you prefer 😉

**Pull request closed without merging**